Events Page

Nate41223 opened the pull request:
> The events page has been made. There is a calendar linked through google drive (not sure how we wanted to embed that but i think that good until we decide) and there is a list of events below it.
> this is a rough draft and i need suggestions on both layout and code. I don't feel like I built it very effiiciently.

Nate41223 committed:
Events Page

> The events page has been made. There is a calendar linked through google drive (not sure how we wanted to embed that but i think that good until we decide) and there is a list of events below it.
> this is a rough draft and i need suggestions on both layout and code. I don't feel like I built it very effiiciently.

SweaterWeather commented:
1.  Nav bar displays behind the page content, please order the site contents to display behind the nav bar.
(CSS)

2. Date information is stored within a figcaption element, please store date information inside of a time element (time datetime="2008-02-14 20:00").
(line 29, line 39, line 49)

3. Divs are styled using div style="clear: both;" in HTML, please use CSS to style page elements.
(line 34, line 44, line 54)

molliehettinger commented:
Design wise, taking out the underline on Events would make it feel a little bit messy. Separate the events from the title a bit more. Bolden the Upcoming Events header to have it draw the eye. Match your font color to Nick's original page's text colors.

Nate41223 committed:
Fixed based upon pull request comment

> changed the z-index in the main.css page to 1 so that it takes priority over everything else on the page

> changed all figcaption elements to time elements since they were holding times origionally

> removed divs using "clear: both;" that were creating spacing between current events. it was not necessary  anymore based on current build

Nate41223 committed:
aesthetic changes

> resized the event images to 200 by 200
> changed event images to be a clickable link to take users to the event page
> resized the "upcoming events" title to 45px
> resized the "upcoming events title" title to 30px
> made the "upcoming events title" title to be a clickable link to take users to the event page
> resized the paragraphs to 15px
> changed paragraph color to #434344 to match the other pages
> adjusted margins accordingly to fit the resizing

Nate41223 committed:
editted margins

> changed the top margin of the div element surrounding the events to 45px from 56px so that it matches the individual event spacing
> changed the margin of the ul surrounding the events to 0. This was not causing any visible changes currently but could down the road if we change the layout

nickworks committed:
Merge branch 'master' into Stenman_EventsPage
